### PR TITLE
Ft job filter 165274294

### PR DESCRIPTION
**Pull request by CeciliaCaroline**
#### What does this PR do?
Implements the filtering of jobs by gender

#### How should this be manually tested?
- Pull the branch
- Run the following commands
`yarn run relay`
`yarn start`

#### What are the relevant pivotal tracker stories?
- [#165274294](https://www.pivotaltracker.com/story/show/165274294)

### CODE REVIEW

File: src/components/filter/index.jsx [hunk 1]
```diff
@@ -1,13 +1,57 @@
...
+  Button,
+  FormGroup,
+  Input,
+  Label
+} from "reactstrap";
+import PropTypes from 'prop-types';
+import "./filter.scss";
+
+const Filter = ({ handleGender, gender }) => (
+  <div>
```
Comment (ayz4sci) [inline src/components/filter/index.jsx]: Redundant div

**Review by ayz4sci — APPROVED** [review]
LGTM

File: src/components/filter/filter.scss [hunk 2]
```diff
@@ -6,8 +6,9 @@
 .checkbox{
     margin: 0px;
 }
+
 .list-item{
     display: flex;
     justify-content: space-between;
     width: 110px;
-}
+}
```
Comment (dotNesh) [inline src/components/filter/filter.scss:14]: Leave a line at the end of this file.

**Review by dotNesh — COMMENTED** [review]
Looks good to me. Ensure to leave a line at the end of every file.

**Review by kisakyegordon — APPROVED** [review]
Looks good to me

File: src/components/home/index.jsx [hunk 3]
```diff
@@ -11,28 +12,39 @@ import './home.scss';
...
+    const { searchTerm, gender } = this.state;
+
     const DashboardQueryRenderer = (
       <QueryRenderer
         environment={environment}
         query={graphql`
-          query homeQuery ($searchTerm: String) {
-            allJobs(jobTitle_Icontains: $searchTerm, before: "12/3/2020") {
+
+          query homeQuery ($searchTerm: String, $gender: String) {
```
Comment (dotNesh) [inline src/components/home/index.jsx:37]: The name tag for this query should be `HomeQuery`. Relay expects the tag name to begin with the module name and end with the action(Mutation, Query, Subscription).

**Pull request merged**